concreate/builders/osbs.py

```diff
@@ -171,9 +171,9 @@ def add(self):
         # Add new Dockerfile
         subprocess.check_call(["git", "add", "Dockerfile"])
 
-        for d in ["repos", "modules"]:
-            if os.path.exists(os.path.join(self.output, d)):
-                subprocess.check_call(["git", "add", d])
+        for d in ["repos", "modules", "ASAS"]:
```

Comment:
"ASAS"?

Reply:
I've testes something aleft it here, fixing